xdsl/dialects/gpu.py

```diff
@@ -71,37 +71,31 @@ class _GPUAttr(ParametrizedAttribute, Generic[T]):
     def parse_parameters(parser: Parser) -> list[Attribute]:
         parser.parse_characters(
             "<",
-            f"Expected <. gpu attributes currently have the #gpu<name value> syntax.",
+            ": gpu attributes currently have the #gpu<name value> syntax.",
```

Comment:
Why is the colon necessary here? Shouldn't that be added on the other side of the string format?

Reply:
I guess it's a style choice we can change.
Currently, parser methods with a `context_msg` option are printed like this: `"error-msg" + context_msg`, like in MLIR.
So, we can have error messages like `'<' expected in attribute parameters` for instance, making it a bit more concise and readable I think.